projects/knora-ui/src/lib/viewer/values/list-value/list-value.component.html

```diff
@@ -23,23 +23,28 @@
         </span>
     </mat-menu>
 </span>
-
-<span [formGroup]="form">
-  <mat-form-field class="large-field child-value-component" *ngIf="mode === 'read'" floatLabel="never">
-        <input [formControlName]="'listValue'" class="value" type="text" placeholder="List value" matInput [readonly]="true">
-  </mat-form-field>
-  <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-    <textarea matInput
-              cdkTextareaAutosize
-              #autosize="cdkTextareaAutosize"
-              cdkAutosizeMinRows="1"
-              cdkAutosizeMaxRows="6"
-              [formControlName]="'comment'"
-              class="comment"
-              placeholder="Comment"
-              type="text"
-              [readonly]="mode === 'read'"
-              spellcheck="false">
-    </textarea>
-  </mat-form-field>
-</span>
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <p class="rm-value">{{valueFormControl.value}}</p>
+    <p class="rm-comment" *ngIf="shouldShowComment">{{commentFormControl.value}}</p>
+</div>
+<ng-template #showForm>
+    <span [formGroup]="form">
+    <mat-form-field class="large-field child-value-component" *ngIf="mode === 'read'" floatLabel="never">
+            <input [formControlName]="'listValue'" class="value" type="text" placeholder="List value" matInput [readonly]="true">
```

Comment:
see comment about readonly.

Reply:
Will be done in https://dasch.myjetbrains.com/youtrack/issue/DSP-206